Exiled.Events/EventArgs/LandingEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="LandingEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs
+{
+    using System;
+    using Exiled.API.Features;
+
+    /// <summary>
+    /// Contains all the information when a player lands.
```

Comment:
/// Contains all the information when a player lands **on the ground**
It isn't really clear what Landing means, I mean, there aren't much possibilities of it but make it clear.

Reply:
It's an old version, now it's different.